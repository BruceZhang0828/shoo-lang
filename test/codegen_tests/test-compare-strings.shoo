string z = "hijjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj";
string q = "hijjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj";

if (q == z) {
    println("strings are equal");
} else {
    println("strings aren't equal");
}

function something(string s) string {
    string h = "h";
    string j = h + s;
    return j;
}

string p = something("ijjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjjj");
if (p == z) {
    println("strings are equal");
} else {
    println("strings aren't equal");
} 
