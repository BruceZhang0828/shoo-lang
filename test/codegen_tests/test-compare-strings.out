strings are equal
strings are equal
